examples/run-benchmarks.sh

```diff
@@ -95,58 +95,78 @@ BPGDEC="$(which bpgdec)"
 # edit below to provide the path to the chosen version of VTM
 AV1_BIN_DIR="${HOME}/av1/aom/build_gcc"
 
+gen_qps(){
+    if [ $# -eq 2 ]; then
+        qps=`echo $(seq $1 $2) | awk '{gsub(/ /, ",");print $0}'`
+    elif [ $# -eq 3 ]; then
+        qps=`echo $(seq $1 $2 $3) | awk '{gsub(/ /, ",");print $0}'`
+    else
+        echo "gen_qps: invalid number of arguments"
+        exit 1
+    fi
+    echo $qps
+}
+
 jpeg() {
+    qps=`gen_qps 5 5 95`
```

Comment:
I was rerunning some results regarding other issues with bench results, and was waiting to get them to push. I modified this by simply adding the comma like: `$(seq -s, 5 5 95)` 
does your function solve another problem?

Reply:
My function doesn't solve other problems.

Reply:
Same I discovered it when looking up